src/Mobs/Villager.cpp

```diff
@@ -107,6 +107,26 @@ void cVillager::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 
 
+
+void cVillager::KilledBy(TakeDamageInfo & a_TDI)
+{
+	super::KilledBy(a_TDI);
+
+	// TODO: 0% chance on Easy, 50% chance on Normal and 100% chance on Hard
```

Comment:
This is another place where difficulty needs to be moved back into the server from Core.